src/libs/WixToolset.Data/Xsd/wix.xsd

```diff
@@ -8755,6 +8760,22 @@
                         </xs:documentation>
             </xs:annotation>
           </xs:attribute>
+          <xs:attribute name="Architecture">
+            <xs:annotation>
+              <xs:documentation>
+                            Set this attribute to 'always32Bit' to mark this as a 32-bit custom action. Valid only when used with
+                                the Script, VBScriptCall, and JScriptCall attributes. The default value is 'default', which
+                                will be the Architecture set by the -arch switch in candle.exe or the InstallerPlatform property
+                                in a .wixproj MSBuild project. The default value is 'default'.
```

Comment:
I think you can drop the last sentence, you already explained what the default is.